frontend/src/app/containers/App.js

```diff
@@ -70,7 +70,7 @@ class App extends Component {
     if (window.ga && ga.loaded) {
       const data = dataIn || {};
       data.hitType = type;
-      data.page = (pathname === '/') ? pathname : '/' + pathname;
+      data.location = window.location;
```

Comment:
Thoughts on ^ @chuckharmston? Wasn't sure if your `measurePageView` suggestion would still be valid knowing that we have to use the entire location field.